apps/argocd.py

```diff
@@ -50,7 +50,7 @@ def objects():
                     "oidc.config": yaml.dump(
                         {
                             "name": "Keycloak",
-                            "issuer": "https://auth.ocf.berkeley.edu/auth/realms/ocf",
+                            "issuer": "https://idm.ocf.berkeley.edu/realms/ocf",
```

Comment:
When deploying this, be careful to not sync argocd until we're sure the new keycloak works! We may even want to push this change to a future PR.

Reply:
it's already deployed, it works

Reply:
(deployed for testing purposes)